tests/src/ExistingSite/FrontPageHeadingTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\oe_showcase\ExistingSite;
+
+/**
+ * Class to test the presence of H1 on the front page.
+ */
+class FrontPageHeadingTest extends ShowcaseExistingSiteTestBase {
```

Comment:
```suggestion
class FrontPageTest extends ShowcaseExistingSiteTestBase {
```
lets name the class more generic, we might add more stuff in the future, the method name can stay the same

Reply:
done.